prow/pj-on-kind.sh

```diff
@@ -25,29 +25,34 @@ function main() {
 
   # Generate PJ and Pod.
   mkpj "--config-path=${config}" "${job_config}" "--job=${job}" > "${PWD}/pj.yaml"
-  mkpod --build-id=snowflake "--prow-job=${PWD}/pj.yaml" --local "--out-dir=/output/${job}" > "${PWD}/pod.yaml"
+  mkpod --build-id=snowflake "--prow-job=${PWD}/pj.yaml" --local "--out-dir=${out_dir}" > "${PWD}/pod.yaml"
```

Comment:
might be better to write to a tempdir instead of PWD?

Reply:
I used a tempdir initially, but I switched to using PWD so that the pj.yaml and pod.yaml can be easily inspected and modified. WDYT?

Reply:
Hmm yeah. I tend to avoid writing to PWD in tools but...

You could also tee them to the shell?

Reply:
> You could also tee them to the shell?

Yeah that would work, but seems a little less convenient. pod.yaml and pj.yaml are in the .gitignore for `config/` and `prow/` at least so I think this is probably ok for now.

Reply:
I think that tee would result in too much stdout clutter.

I would prefer it if it wrote to a tempdir and echoed something like `Created foo/dir.yaml`. It's a nit, but this way I don't have to delete them to get them off my `git status` readout.

Reply:
I fixed the `.gitignore` so that the files will be ignored anywhere in the repo. Echoing the temp dir could work, but switching directories every time pj-on-kind.sh is run would be annoying when trying to make iterative changes to the pod.yaml and pj.yaml.  It would also make it harder to modify the files (currently you can just comment out the file generation to use your own pj.yaml and pod.yaml, but using a temp dir would make this harder)